src/core/common/bit_vector.hpp

```diff
@@ -95,11 +104,7 @@ template <uint16_t N> class BitVector : public Equatable<BitVector<N>>
      * @param[in] aIndex  The index.
      *
      */
-    void Clear(uint16_t aIndex)
-    {
-        OT_ASSERT(aIndex < N);
-        mMask[aIndex / 8] &= ~(0x80 >> (aIndex % 8));
-    }
+    void Clear(uint16_t aIndex) { Set(aIndex, false); }
```

Comment:
How about removing `Clear()` in light of now having `Set(aIndex, false)`?
I think it'd good to provide one way for this.

Reply:
We can let BitVector inherit Clearable again and use Clear(void) instead of ClearAll(void). Thoughts?

Reply:
Will resolve it in a separate PR.